src/ColorPicker/index.jsx

```diff
@@ -3,14 +3,12 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { HexColorPicker } from 'react-colorful';
 
-import {
-  Button,
-  Form,
-  ModalPopup,
-  OverlayTrigger,
-  Tooltip,
-  useToggle,
-} from '..';
+import Button from '../Button';
+import Form from '../Form';
+import ModalPopup from '../Modal/ModalPopup';
+import { OverlayTrigger } from '../Overlay';
+import Tooltip from '../Tooltip';
+import useToggle from '../hooks/useToggle';
```

Comment:
[clarification] Are these import changes (and others) due to a linting rule? Just curious :)

Reply:
yeah, after updating `eslint` version and running `npm run lint` I got like 60 errors about circular imports... That was because in components' files we often imported other components from root `index.js` file, so we fixed that. (I'm pretty sure it didn't cause any issues with the build, but still it's awesome `eslint` started flagging such cases)